src/PrefillGravityForms/Controllers/WeAreFrankController.php

```diff
@@ -99,9 +101,17 @@ protected function getExpandFields(string $expand): array
         return array_filter(explode(',', $expand));
     }
 
-    protected function fetchPersonData(array $preparedData): array
+    /**
+     * @inheritDoc
+     */
+    protected function extractBSN(array $response): string
+    {
+        return (string) ($response['personen'][0]['burgerservicenummer'] ?? '');
```

Comment:
Hier ook de vraag: Waarom willen we allerlei omstandigheden als missende keys en non-string BSNs afvangen? Verwachten we die?